### PR TITLE
GUI update

### PR DESCRIPTION
**Pull request by murgui**
Added all changes listed in Changes to GUI.
Missing to consider rotor speed.

### CODE REVIEW

**Comment by murgui** [comment]
Everything done, ready to roll!

File: Software/Development/SemesterProject/App/communicationManager.c [hunk 1]
```diff
@@ -120,8 +143,8 @@ void getAnalogSignals(void)
 //    COMMS_PCBBrakeReference = getBrakeReferenceSliderMeasurement();
     COMMS_PedalTorqueReference = getTorqueReferencePedalMeasurement();
     COMMS_PedalBrakeReference = getBrakeReferencePedalMeasurement();
-    COMMS_Thermometer1 = getThermometer1Measurement();
-    COMMS_Thermometer2 = getThermometer2Measurement();
+    COMMS_Thermometer1 = 100 * getThermometer1Measurement();
```
Comment (NicolaiFransen) [inline Software/Development/SemesterProject/App/communicationManager.c]: Use the define from constants.h

**Review by NicolaiFransen — APPROVED** [review]
¿Ship it?

**Review by AitorTeran — APPROVED** [review]
Nothing to say! looking good.
Ship it!¡¡!

File: Software/Development/SemesterProject/App/communicationManager.c (hunk 1 above)
Comment (murgui) [reply Software/Development/SemesterProject/App/communicationManager.c]: you are better than that

**Pull request merged**